crates/config/src/lib.rs

```diff
@@ -2275,16 +2275,16 @@ impl Default for Config {
             fs_permissions: FsPermissions::new([PathPermission::read("out")]),
             isolate: cfg!(feature = "isolate-by-default"),
             root: root_default(),
-            src: "src".into(),
-            test: "test".into(),
-            script: "script".into(),
-            out: "out".into(),
-            libs: vec!["lib".into()],
+            src: "assertions/src".into(),
+            test: "assertions/test".into(),
+            script: "assertions/script".into(),
+            out: "assertions/out".into(),
+            libs: vec!["assertions".into()],
```

Comment:
is this correct?

Reply:
[3420f4c](https://github.com/phylaxsystems/phoundry/pull/38/commits/3420f4cb98bb07438b97d3b3c565d61441ae37cb)